client/src/components/FollowersView/FollowersView.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import UserCard from 'components/UserCard/UserCard'
+import './FollowersView.css'
+
+function FollowersView({
+  profile,
+  followers
+}) {
+  const users = followers ? profile.followers : profile.following
+  return (
+    <div className="followers-view">
+      <div className="playlist-container">
+        <div className="header">
+          <h3>{followers ? `${profile.username}'s Followers` : 'Following'}</h3>
+        </div>
+        <div className="playlists">
+          {users.length !== 0 ? (
```

Comment:
I think you're overusing the ternary operator a bit. I recommend really only using it for single line statements and then using if/else for multi-line statements.

Reply:
I don't think I can use if/else statements in jsx. 
[https://react-cn.github.io/react/tips/if-else-in-JSX.html#:~:text=if%2Delse%20statements%20don't,function%20calls%20and%20object%20construction.](url)